Find a functionally equivalent code:android.media.MediaPlayer.TimeProvider.onStopped:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -3,8 +3,10 @@
         if (DEBUG)
             Log.d(TAG, "onStopped");
         mPaused = true;
+        mStopped = true;
+        mSeeking = false;
         scheduleNotification(NOTIFY_STOP, 0);
     }
 }

